config/runtime.exs

```diff
@@ -35,6 +35,9 @@ end
 
 sentry_dsn = System.get_env("SENTRY_DSN")
 
+api_key = System.get_env("SCREENPLAY_API_KEY")
+if config_env() != :test and api_key in [nil, ""], do: raise("SCREENPLAY_API_KEY key not set")
```

Comment:
To avoid subtle differences in behavior between environments, I think it'd be better to add a test API key to the CI env and have this always be required.

We could also use `System.fetch_env!` here, though that would allow a set-but-empty API key, if we have reason to believe that could happen unintentionally.

Reply:
That's a good idea. The scenario I imagine for set-but-empty is in local development, and I'm not super concerned with that. TF enforces values to exist in deploy environments.